mozetl/main.py

```diff
@@ -41,17 +41,17 @@ def etl_job(sc, sqlContext):
     results = transform_pings(get_data(sc))
 
     # Display results:
-    total = sum(map(operator.itemgetter(1), results.iteritems()))
+    total = sum(map(operator.itemgetter(1), iter(results.items())))
     # Print the OS and client_id counts in descending order:
     sorted_results = sorted(
-        results.iteritems(),
+        iter(results.items()),
         key=operator.itemgetter(1),
         reverse=True,
     )
     for pair in sorted_results:
-        print(
+        print((
             "OS: {:<10} Percent: {:0.2f}%".format(
                 pair[0],
                 float(pair[1]) / total * 100
             )
-        )
+        ))
```

Comment:
It doesn't, but it's probably related to this behavior:

> When the -p is passed, 2to3 treats print as a function instead of a statement. This is useful when from __future__ import print_function is being used. If this option is not given, the print fixer will surround print calls in an extra set of parentheses because it cannot differentiate between the print statement with parentheses (such as print ("a" + "b" + "c")) and a true function call.

https://docs.python.org/3.0/library/2to3.html

I can see if I can minimize this, but it occurs frequently enough.

Reply:
To me that reads as adding `-p` should prevent 2to3 from doing this

Reply:
There are some degenerate cases where adding `-p` won't convert the print statement correctly without adding the parenthesis (I ran into this before).

```
RefactoringTool: There were 5 errors:
RefactoringTool: Can't parse mozetl/addon_aggregates/addon_aggregates.py: ParseError: bad input: type=3, value=u'"loading..."', context=(u' ', (68, 10))
RefactoringTool: Can't parse mozetl/hardware_report/hardware_dashboard.py: ParseError: bad input: type=3, value=u'"Joining JSON files..."', context=(u' ', (51, 10))
RefactoringTool: Can't parse mozetl/tab_spinner/generate_counts.py: ParseError: bad input: type=3, value=u'"Start Date: {}, End Date: {}"', context=(u' ', (24, 10))
RefactoringTool: Can't parse mozetl/tab_spinner/utils.py: ParseError: bad input: type=3, value=u'"Short and longcounts match"', context=(u' ', (156, 14))
RefactoringTool: Can't parse tests/test_addon_aggregates.py: ParseError: bad input: type=1, value=u'first_install', context=(u' ', (151, 14))
```
I can manually fix these up to reduce the noise though.